src/components/App.js

```diff
@@ -25,6 +25,8 @@ const App = () => {
     if (!board.boardId) {
       dispatch(setNewBoardState());
     }
+
+    document.getElementById("lists-wrapper").scrollLeft = 0;
```

Comment:
I've used a `document.getElementById` here as the div already has a ref specified which is needed for the drag and drop functionality